pl_examples/domain_templates/generative_adversarial_net.py

```diff
@@ -8,12 +8,11 @@
 """
 import os
 from argparse import ArgumentParser, Namespace
-from collections import OrderedDict
 
 import numpy as np
 import torch
 import torch.nn as nn
-import torch.nn.functional as F
+import torch.nn.functional as F  # noqa
```

Comment:
is it used somewhere?

Reply:
If you mean `F` yes. In the loss function. If you mean `#noqa` it is to stop my linter from bothering me :)

https://github.com/kureta/pytorch-lightning/blob/24ec0704129032ddf3090fe6904815ca4ea0f47f/pl_examples/domain_templates/generative_adversarial_net.py#L107